src/app/submission/sections/upload/section-upload.component.html

```diff
@@ -26,16 +18,26 @@ <h3 class="text-center"><span class="text-muted">{{'submission.sections.upload.n
       </ds-alert>
     </div>
   </div>
-
-  <ng-container *ngFor="let fileEntry of fileList">
+  <div class="row">
+    <div class="col-md-2">
+      <span class="text-left font-weight-bold">{{ 'bitstream.edit.form.primaryBitstream.label' | translate }}</span>
```

Comment:
I find the layout of the column to be odd.  It's difficult to tell this is a table (see screenshot).  Should we make sure the files also have a header?  Perhaps make it like this?
```
----------------------------------------------
 Primary File   |    File Details
----------------------------------------------
```

NOTE: I specifically changed the heading to "primary file" as this is called the "Upload files" section.  I'm not sure if users will all understand "primary bitstream"? 

If you feel it should be called "primary bitstream" on the submission form, we can do so.  I'm just not sure if all users will understand that bitstreams and files mean the same thing.